mantidproject/mantid
# Show the SliceSelector widget when invoquing the DynamicPDF custom-interface

Fixes #14936
![14936_sliceselector](https://cloud.githubusercontent.com/assets/1136006/12986546/d5cb45be-d0c5-11e5-9065-f17a88632750.png)

Testing:
1. compile
2. open the interface (Interfaces->Dynamic PDF->Dynamic PDF Slice Selector
3. Load a matrix workspace
4. Verify the 2D view shows the workspace
